libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -1,24 +1,36 @@
 <ng-container *ngIf="(readingList$ | async).length > 0; else empty">
-  <section class="reading-list-item" *ngFor="let readingItem of readingList$ | async">
-    <div>
-      <img class="reading-list-item--cover" [src]="readingItem.coverUrl" alt="" />
-    </div>
-    <div class="reading-list-item--details">
-      <strong class="reading-list-item--details--title">{{ readingItem.title }}</strong>
-      <div class="reading-list-item--details--author">
-        {{ readingItem.authors.join(',') }}
+  <section *ngFor="let readingItem of readingList$ | async">
+    <div class="reading-list-item">
+      <div>
+        <img class="reading-list-item--cover" [src]="readingItem.coverUrl" alt="" />
```

Comment:
Should this `alt` attribute be there if it's empty? And since it's there, should it be empty?

Reply:
This was changed as part of first task, Added empty alt attribute for decorative image (as the text shown in image is already presented to its right) - Referred from https://www.w3.org/WAI/tutorials/images/decision-tree/